docs/topics/forms/modelforms.txt

```diff
@@ -1311,6 +1311,51 @@ Then when you create your inline formset, pass in the optional argument
     >>> author = Author.objects.get(name="Mike Royko")
     >>> formset = BookFormSet(instance=author)
 
+.. _inline-formset-interaction-with-prefetching:
```

Comment:
`get_instance_by_queryset_index()` should also be documented under `models.BaseModelFormSet` [docs](https://docs.djangoproject.com/en/5.0/topics/forms/modelforms/#model-formsets). Is it useful in itself?

Reply:
I'm not sure how to mention `get_instance_by_queryset_index()` in the 'Model formsets' section.
I can't think of a reason to override that method on it's own.